src/btree/bt_cursor.c

```diff
@@ -26,19 +26,17 @@ typedef struct {
  */
 static int
 __btcur_bounds_contains_key(WT_SESSION_IMPL *session, WT_CURSOR *cursor, WT_ITEM *key,
-  uint64_t recno, bool *key_out_of_boundsp, bool *upperp)
```

Comment:
Is there a reason why we choose to use a pointer to the recno, rather than just the recno number itself?

Reply:
That's a good point, in theory it helped when I was pulling them off the cursor but it would be better to instead pass RECNO_OOB instead of a NULL pointer. I'll make the change.

Reply:
[954d737](https://github.com/wiredtiger/wiredtiger/pull/8201/commits/954d73783964eaf3285061ad0c20ebaaccd39a95)